aws/sagemaker-pytorch-inference-toolkit
# infra: Make docker folder read only, remove unused tests, update reponame, remove incorrect documentation.

* Make docker folder read only
* Remove unused tests
* Rename test-toolkit/ -> test/
* Rename buildspec.yml -> buildspec-container-pr.yml 
* Rename buildspec-toolkit.yml -> buildspec.yml 
* Update repo name
* Remove incorrect documentation.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
